src/bun.js/bindings/bindings.cpp

```diff
@@ -4114,6 +4115,9 @@ static JSC::Identifier builtinNameMap(JSC::JSGlobalObject* globalObject, unsigne
     case BuiltinNamesMap::headers: {
         return clientData->builtinNames().headersPublicName();
     }
+    case BuiltinNamesMap::statusText: {
+        return clientData->builtinNames().statusTextPublicName();
+    }
```

Comment:
all of the bindings{.cpp/.zig} changes were very much dogscience.  are there tests or things I should worry about?

Reply:
this is right, thanks

context here: `Identifier::fromString` is super slow. It frequently shows up in profiles. It has a lock, it clones a string and hashes it each call